els/frontend/src/components/layout/Header.js

```diff
@@ -64,10 +84,14 @@ export class Header extends Component {
           </button>
           <div className="collapse navbar-collapse" id="navbarTogglerDemo01">
             <a className="navbar-brand" href="#">
-              E-Learning System | ADMIN
+              {!isAuthenticated
+                ? 'Welcome to E-Learning System'
+                : `${user.is_staff}` == 'true'
```

Comment:
```suggestion
                user.is_staff ? 'E-Learning System | ADMIN' : 'E-Learning System'
```

@JohnRoyCabezas we can just directly write is like this since `is_staff` is boolean right? Please change this.

Reply:
@roseaugusto Oh wow I forgot to take that into account. Thanks a lot. The code looks more cleaner now.